docs/design/admission_control_limit_range.md

```diff
@@ -73,8 +73,8 @@ type LimitRangeItem struct {
   Min ResourceList `json:"min,omitempty" description:"min usage constraints on this kind by resource name"`
   // Default resource limits on this kind by resource name
   Default ResourceList `json:"default,omitempty" description:"default resource limits values on this kind by resource name if omitted"`
-  // DefaultRequests resource requests on this kind by resource name
-  DefaultRequests ResourceList `json:"defaultRequests,omitempty" description:"default resource requests values on this kind by resource name if omitted"`
+  // DefaultRequest resource requests on this kind by resource name
+  DefaultRequest ResourceList `json:"defaultRequest,omitempty" description:"default resource requests values on this kind by resource name if omitted"`
```

Comment:
At the time, it bothered me that we were not consistent with the rest of the object.

For example, I thought if we have DefaultRequests, we should also have Defaults.

Since we have Default now, I thought DefaultRequest made sense.